app/guides/layout.tsx

```diff
@@ -0,0 +1,24 @@
+"use server";
+
+import { getGuidePages } from "@/lib/server/cms-data";
+import { getSections } from "@/lib/nav-menu-builder";
+import NavMenu from "@/components/NavMenu";
+
+export default async function GuidesLayout({
+  children,
+}: {
+  children: React.ReactNode;
+}) {
+  const guidePages = await getGuidePages();
+
+  return (
+    <>
+      <div className="flex flex-col items-center">
+        <div className="flex flex-col items-start w-[min(100%)] lg:w-[min(100%,1024px)] pl-2 sm:pl-0">
```

Comment:
you could try using the `container` class from tailwind.

and then `min-w-full`

Reply:
`container` is a good call here, however `min-w-full` breaks it as that changes it to use the whole page.

![image](https://github.com/user-attachments/assets/f76fcc64-2f8a-4ae9-84b5-1cb85505e3e8)
![image](https://github.com/user-attachments/assets/e87631cf-42cb-4afb-9fde-68f62199aa2f)